controllers/fetchArticleByArticleIdController.js

```diff
@@ -0,0 +1,15 @@
+const { getArticleByArticleId } = require('../models/getArticleByArticleIdModel.js')
+
+const fetchArticleByArticleId = (req, res, next) => {
+    const { article_id } = req.params;
+
+    getArticleByArticleId(article_id).then((articles) => {
```

Comment:
The response should be a single object so this arguement name should be singular.